README.md

```diff
@@ -40,7 +40,6 @@ Finally, this implementation uses the [ASP.NET Core Docker sample web app](https
 - [Flux GitOps Operator](https://fluxcd.io)
 - [Traefik Ingress Controller](https://doc.traefik.io/traefik/v2.4/routing/providers/kubernetes-ingress/)
 - [Azure AD Pod Identity](https://github.com/Azure/aad-pod-identity)
-- [Azure KeyVault Secret Store CSI Provider](https://github.com/Azure/secrets-store-csi-driver-provider-azure)
```

Comment:
Why is this removed? We still use it. Can you instead link to the docs page? While you're here, can you do the same for the line above?